docs/team/daryl.md

```diff
@@ -0,0 +1,27 @@
+# Daryl's Project Portfolio Page
+
+## Project: WorkBook
+
+### Overview
+
+WorkBook (WB) is a desktop app for CS/tech students who are applying for internships to manage their internship applications, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, WB can get your internship management tasks done faster than traditional GUI apps.
+
+### Summary of contributions
+
+#### Code contributed
+This displays my code contributions for the project: [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=rylzxc&breakdown=true)
+
+#### Enhancements implemented
+
+#### Contributions to User Guide
+
+#### Contributions to Developer's Guide
```

Comment:
Nitpick but this should be Developer instead of Developer's

Reply:
Agreed and updated